examples/sw/in.mos2

```diff
@@ -0,0 +1,33 @@
+# bilayer MoS2
+units      	metal
+boundary   	p p f
+processors	* * 1
+
+atom_style 	full
+read_data       ./single_layer_MoS2.data
+
+variable        mS  equal 32.065   # mass of sulphur atom , uint: a.u.=1.66X10^(-27)kg
+variable        mMo equal 95.94    # mass of molebdenum atom , uint: a.u.=1.66X10^(-27)kg
+
+mass            * ${mS}
```

Comment:
Please don't use variables for such trivial cases. That just makes the input harder to read.

Reply:
Fixed.